bin/common.sh

```diff
@@ -197,6 +199,8 @@ function wait_for_resources_to_be_ready() {
 function wait_for_operators_to_be_running() {
   sleep 10 # Workaround for https://github.com/kubernetes/kubernetes/issues/109489
 
+  # oc 4.7.0 doesn't have the oc wait --for=jsonpath=... feature
+  # works with oc 4.11.5
```

Comment:
Since we added oc 4.10+ as a prerequisite, can we remove this comment here since it's irrelevant for people viewing the code? it might also easily become outdated.